src/main/java/app/models/personRegistration/PersonRegistration.java

```diff
@@ -1,4 +1,4 @@
-package app.models;
+package app.models.personRegistration;
```

Comment:
Package name should be in lower case accodring to java conversions. It's also better to have `person.registration` package instead of a single `personregistration`